### PR TITLE
67667 Adds unit test coverage for DetailsVA component

### PR DESCRIPTION
**Pull request by ryanshaw**
## Summary

This increases test coverage for the the [`DetailsVA`](https://github.com/department-of-veterans-affairs/vets-website/blob/611c2541d5e5f22449bfe149f161b52e523ccf73/src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVA.jsx) component within VAOS.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/67667

## Testing done
- e2e testing
- Unit testing

## Acceptance criteria

- [ ] The test coverage should be at 100% for this component:
image

- [ ] New tests are correct and appropriate

### CODE REVIEW

**Comment by ryanshaw** [comment]
I'm still in the process of adding `defautProps` to this.

**Review by vbahinwillit — APPROVED** [review]

**Review by ataker — APPROVED** [review]

**Review by simiadebowale — COMMENTED** [review]
@ryanshaw  - can we add a check for the PrintLink component?

**Comment by ryanshaw** [comment]
@simiadebowale I cleaned up the test a bit and also added coverage for `PrintLink`

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVA.jsx [hunk 1]
```diff
@@ -120,7 +116,59 @@ export default function DetailsVA({ appointment, facilityData }) {
...
+      stationId: PropTypes.string.isRequired,
+      clinicName: PropTypes.string.isRequired,
+    }),
+  }),
+  facilityData: PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    vistaId: PropTypes.string.isRequired,
+    name: PropTypes.string.isRequired,
+  }),
   useV2: PropTypes.bool,
```
Comment (vbahinwillit) [inline src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVA.jsx:145]: Is 'useV2' still used?

**Review by vbahinwillit — APPROVED** [review]

**Pull request merged**